wp-content/themes/access/src/js/modules/screener.js

```diff
@@ -75,6 +78,33 @@ class Screener {
       e.preventDefault();
       const hash = window.location.hash;
       const $section = $(hash);
+      let title = $section.find('[data-js="question"]');
+      let prevStep;
+
+      this._history.push(hash);
+
+      if (this._history.length > 1) {
+         prevStep = this._history[this._history.length - 2];
+      }
+
+      let stateObj = {
+        prevStep: prevStep,
+        step: `${hash}`,
```

Comment:
Template interpolation is not really needed here, you can just use `hash` instead of ``${hash}``